src/app/shared/Note.svelte

```diff
@@ -102,11 +104,21 @@
       .open()
 
   const removeFromContext = e => {
+    console.log("remove from context", e)
     context = reject(whereEq({id: e.id}), context)
   }
 
-  const addToContext = events => {
-    context = context.concat(events)
+  const addDraftToContext = (event, cb) => {
+    event.status = "draft"
+    event.remove = () => {
+      cb()
+      removeFromContext(event)
+    }
+    context = context.concat(event)
```

Comment:
Mutating events isn't a good idea. I don't think it will technically break anything, but all this stuff gets serialized and shoved into indexeddb, which will get weird fast. See if you can find another way to pass this data between components.

Reply:
Yeah I tried, but it's difficult because Note is used recursively, having the event state and functionality on the event object simplified a lot of stuffs.
Aren't those event already pulled and serialised from indexedDB?